utils/java/mvn.go

```diff
@@ -83,9 +83,6 @@ func (mdt *MavenDepTreeManager) RunMavenDepTree() (depTreeOutput string, clearMa
 	// depTreeExecDir is a temp directory for all the files that are required for the maven-dep-tree run
 	depTreeExecDir, clearMavenDepTreeRun, err := mdt.CreateTempDirWithSettingsXmlIfNeeded()
```

Comment:
can CreateTempDirWithSettingsXmlIfNeeded be private? I can see its only usage at core is at mvn.go?

Reply:
it is also used in mavenPackageHandler to avoid the execution of the entire plugin